arch/arm64/src/common/arm64_fatal.c

```diff
@@ -527,10 +527,10 @@ static int arm64_exception_handler(struct regs_context *regs)
 
   if (ret != 0)
     {
-      sinfo("CurrentEL: %s\n", el_str);
-      sinfo("ESR_ELn: 0x%" PRIx64 "\n", esr);
-      sinfo("FAR_ELn: 0x%" PRIx64 "\n", far);
-      sinfo("ELR_ELn: 0x%" PRIx64 "\n", elr);
+      serr("CurrentEL: %s\n", el_str);
```

Comment:
You could even change the most critical messages to _alert(), like which exception occurred.

Also, ESR, FAR and ELR are absolutely vital information when taking a fatal exception where you absolutely CANNOT trust the stack contents (integer context).

Reply:
done